crates/sui-indexer-alt-graphql/src/api/types/event/filter.rs

```diff
@@ -173,6 +169,20 @@ impl EventFilter {
     }
 }
 
+impl TxBoundsFilter for EventFilter {
```

Comment:
`TxBoundsFilter` does not make much sense as a name here, given the functions are all about checkpoints, it seems like this trait is being name for how it is used (to calculate bounds of transaction sequence numbers) rather than what it is -- what about `CheckpointBounds`?